lib/stackConfiguration/src/main/java/org/sagebionetworks/StackConfigurationImpl.java

```diff
@@ -786,15 +786,15 @@ public String getOAuth2GoogleClientSecret() {
 	 * @return
 	 */
 	public String getOAuth2ORCIDClientId() {
-		return stackEncrypter.getDecryptedProperty("org.sagebionetworks.oauth2.orcid.client.id");
+		return stackEncrypter.getDecryptedProperty("org.sagebionetworks.oauth2.orcid.client.id.v2");
```

Comment:
Will update the stack builder: Synapse-Stack-Builder\src\main\resources\templates\repo\repo-defaults.properties